doc/index.rst

```diff
@@ -21,6 +21,7 @@ Contents
    :maxdepth: 2
 
    installation
+   cli
```

Comment:
Personally I'd put this below API but I don't have a strong opinion.

Reply:
When I started using signac, I think I used the command line more for daily tasks before using signac in scripts, so based on "order of learning" maybe CLI can be first.